src/components/forms/ListBuilder.jsx

```diff
@@ -32,8 +32,9 @@ const ListBuilder = ({ fieldsList, locale, hasShowButton, resource }) => {
               return (
                 <ReferenceField
                   key={index}
-                  source={field.name}
+                  source={field.name + "Id"}
```

Comment:
please make sure other lists are still working

Reply:
Right now it's only used for the employee list, so it's working just fine. The other way would be to revert this line and change line 152 on employees.js to `{ name: "countryId", type: "reference", reference: "countries" }`
which one do you think it's better?